conf/trusts.routes

```diff
@@ -8,4 +8,4 @@ POST        /tax-enrolment/callback/subscriptions      uk.gov.hmrc.trusts.contro
 
 GET         /:utr                                      uk.gov.hmrc.trusts.controllers.GetTrustController.get(utr: String)
 
-POST        /variation                                 uk.gov.hmrc.trusts.controllers.VariationsController.variation
+POST        /trustvariation                            uk.gov.hmrc.trusts.controllers.TrustVariationsController.trustVariation
```

Comment:
The endpoint should just be `/variation`

Reply:
removed the trustvariation and changed to variation